.github/actions/run-monitored-tmpnet-cmd/action.yml

```diff
@@ -110,4 +111,3 @@ runs:
       with:
         name: ${{ inputs.artifact_prefix }}-kind-logs
         path: /tmp/kind-logs
```

Comment:
Benchmark does not produce logs in the same location, so we must allow no files to be found to re-use it. If we are going to re-use for this purpose, we should consider how to split out this functionality or just add an input boolean to determine whether to attempt to load artifacts at this location.